components/atoms/AlphaBanner.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * Displays the Alpha AlphaBanner on the page
+ */
+
+export const AlphaBanner = ({ text }) => {
+  return (
+    <div className="bg-gray-100">
+      <div className="lg:container xxs:mx-0 xxs:px-0 lg:mx-auto lg:px-0 xxl:mx-auto">
+        <div className="xxs:w-screen lg:w-2/3 xl:w-1/2 p-4">
+          <div className="flex">
+            <span className="max-h-9 uppercase border-2 border-black p-1 tracking-widest ml-2">
+              Alpha
+            </span>
+            <span className="break-words ml-4 pt-2">{text}</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+AlphaBanner.propTypes = {
+  /**
+   * text area in the AlphaBanner
+   */
+  text: PropTypes.string.isRequired,
+};
```

Comment:
Also, I feel like a better interface for this component would be not to call it an `AlphaBanner` specifically, but a `PhaseBanner`, to pass in the word Alpha as a prop (we can make it the default), and then have it take `children`.

More like this

```
<PhaseBanner phase="Alpha" >This site will change because it's an alpha.</PhaseBanner>
```

Reply:
Font-size has been fixed to go small on mobile view.
padding inside the Alpha tag has been changed to match as close as possible the COVID 19 Benefit Finder. With Tailwind it is hard to get the exact pixel.
The AlphaBanner has been renamed to PhaseBanner. The requested change has been made to pass in children as the text and the phase as the title.

Reply:
Changes addressed